polaris-codemods/tsconfig.json

```diff
@@ -1,13 +1,19 @@
 {
   "extends": "@shopify/typescript-configs/node-base",
   "compilerOptions": {
+    "composite": true,
+    "declaration": true,
+    "declarationMap": true,
+    "incremental": true,
+    "noEmitOnError": true,
+    "skipLibCheck": true,
```

Comment:
TypeScript 5 should fix up this duplication: https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#supporting-multiple-configuration-files-in-extends

`"extends": ["@shopify/typescript-configs/node-base", "../tsconfig.options.json"]`